File: webflux-dialogue/src/main/java/com/study/webflux/rag/domain/voice/port/VoiceSelectionPort.java
package com.study.webflux.rag.domain.voice.port;

import com.study.webflux.rag.domain.dialogue.model.PersonaId;
import com.study.webflux.rag.domain.voice.model.Voice;

public interface VoiceSelectionPort {
	Voice getVoiceForPersona(PersonaId personaId);
}

File: app-miyou/src/main/kotlin/com/miyou/app/domain/voice/port/VoiceSelectionPort.kt
package com.miyou.app.domain.voice.port

import com.miyou.app.domain.dialogue.model.PersonaId
import com.miyou.app.domain.voice.model.Voice

interface VoiceSelectionPort {
    fun getVoiceForPersona(personaId: PersonaId): Voice
}
